src/components/OrderBookWidget.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useMemo } from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0'
+
+import { ORDER_BOOK_HOPS_DEFAULT, ORDER_BOOK_HOPS_MAX } from 'const'
+import OrderBookWidget, { OrderBookProps } from './OrderBookWidget'
+import {
+  defaultNetworkId,
+  defaultBaseToken,
+  defaultQuoteToken,
+  findTokenBySymbolOrAddress,
+  networkMap,
+  tokenSymbols,
+} from 'storybook/tokenDefaults'
+
+const defaultParams: OrderBookProps = {
+  baseToken: defaultBaseToken,
+  quoteToken: defaultQuoteToken,
+  networkId: defaultNetworkId,
+  hops: ORDER_BOOK_HOPS_DEFAULT,
+}
+
+// with price-estimation endpoints
+const availableNetworks = {
+  Mainnet: networkMap.Mainnet,
+  Rinkeby: networkMap.Rinkeby,
+}
+
+export default {
+  title: 'OrderBook/Widget',
+  component: OrderBookWidget,
+  argTypes: {
+    networkId: { control: { type: 'inline-radio', options: availableNetworks } },
+    baseToken: { control: { type: 'select', options: tokenSymbols } },
+    quoteToken: { control: { type: 'select', options: tokenSymbols } },
```

Comment:
Okay, I went for it and solved the conflicts with develop and merged it

It would be good if you rebase develop and add this batchId in your storybook too